### ISSUE TITLE
Board of Directors Experience

### ISSUE THREAD

**Issue by SkyeHoefling** [dotnet-foundation/election#95]
## Question

Working on a Board of Directors comes with its own set of challenges to overcome. These leadership roles differ from organization to organization, while not required I am interested in what leadership experiences candidates are drawing from to help prepare them for a position on the .NET Foundation Board of Directors.

## Background / More Information
The intent of this question is not to prove anyone ineligible but to understand what backgrounds people have in this type of position. If someone doesn't have any formal experience on a board of directors, experience as a sports coach, teacher or mentor are all valid answers in my opinion.

**Comment by jguadagno** [comment]
I've served on the INETA Board of Directors for 8 years as Marketing Director, Vice President, and President.  I've also served on my City (Chandler, AZ, USA) Parks and Recreation Board for 10 years, as a Board Member for 7 years and Chairman for 2.

Over the last 15 years, I have been a Soccer (football), Baseball, Softball coach also.

**Comment by ericsink** [comment]
A few days ago I updated my candidate statement with information along these lines.  I won't repeat all that here.  My previous experience in nonprofit board positions has happened in organizations around religion, education, and social services.

And of course I've been running a software company (with business partners) for over 20 years.

Anyway, I do think you ask an important question. I'm not saying that every one we elect must have deep experience as a board member in other organizations. But the board will be more effective if its members are adept at the role, and previous experience can help with that.

**Comment by Lakritzator** [comment]
Thanks for the question @ahoefling, I like it!
 
I don't have any experience with serving on a board of directors, at least not in an official form, which I'm pretty fine with. I don't have any issue with being responsible and taking decisions for something I have a passion for, and I would be really happy if I am given a chance to learn what it takes to be a good director. If that happens it would be nice if there is someone with experience which I can learn from, but I don't think it's needed, and I will most certainly be a blocking factor.

I had my share, in different roles, of getting people around a table and getting all of them to pull the same strings. I have no problem expressing my thoughts when needed, which I mostly do honest and unfiltered. I really enjoy representing the interests of stakeholders while at the same time realizing we need to be a team and sometimes make compromises.

I have a lot of experience working international, where a lot of different cultures "collide". Even the Dutch and German cultures, being neighbors, have a lot of challenges when doing business with each other. For me it's important that the board of directors has people who can handle these scenarios, and not go into a "director" auto-pilot. Therefor I hope the voters are not all voting for just the experienced people from one country, but make sure there is a healthy mixture.

**Comment by prkhandelwal** [comment]
@ahoefling That's a really great question, as I believe, that being a board member would not only require good technical experience, but also good leadership skills, most important of all, understanding people!

Being a student close to graduation, I have never been in a position to lead in large organizations, but it rather revolves around my school and university life.

I founded Microsoft Student Technical Community in my campus which has recently completed 2 years of operation recently. I held the position of Head of Technology and lead multiple internal project teams.  Our community's overall aim is to spread awareness about Microsoft Technologies among the students.

I am also a Microsoft Student Partner for 2 years now and was recognized as one of the top 100 MSPs among 12 countries for MSP Taiwan summit for our community's efforts and success.

I was also one of the Pioneer member of Young Innovators Club, Johnson Controls in my very first year and later lead a team of YIC members becoming main representative at JCI annual summit.

But most important of all, I did not dot just do these because I had to, but because I loved to!  All these experiences and more have thought me lot about what being a leader and being in a position of such a responsibility is all about. And .NET has been a very integral part of my journey and achievements so far :)

**Comment by ddieruf** [comment]
Hi Andrew, thank you for the question. For non-profits, I have served on 2 boards The Crusade for Children Ambassadors and The Sawyer Hayes Foundation. My leadership roles include starting 2 businesses alone and being a part of over 5 tech start-ups(less than 10 people).

The [Crusade for Children](https://whascrusade.org/) is a local(Louisville, Ky) non-profit that raises money for agencies, schools, and hospitals to help special needs children. Within the Organization there was a need to adopt a younger doner base and one of the long-standing board members recommended myself and another to create a sub-board called the Ambassadors. Our charter was to simply spread awareness of the organization to the 20-35 demographic. We had little money, good backing from the Crusade, and a strong purpose. It was not easy to get going but we did. The Ambassadors grew to 15 people and 3 main events a year, and ultimately a larger young-professionals organization in Lousiville asked to "acquire" the Ambassadors to help drive the cause.

The Sawyer Hayes Foundation was my first board to serve on. The Foundation was a non-profit aiding a very popular State Park(next to my home). It was an odd situation where the Park had state funding but was so popular that there was a separate citizen ran fund as well.  Admittedly I was young and didn't provide too much leadership but aided in all things tech. Social media, web site, etc etc. It was a good introduction for me into how a successful non-profit is run.

Leadership has come naturally to me. I pride myself on being a self-starter and try every day to not be a perfectionist. For me, there is no greater reward than teaching someone from my mistakes/learnings, so they can get ahead a little fast than I did.

As a .NET Foundation board member, I see so many opportunities on the horizon. Some are undefined and need a little creativity to get going, and some need reason and industry knowledge to grow. I welcome these challenges and hope to turn them into success.

**Comment by mitchelsellers** [comment]
I too updated my [candidate profile](https://election.dotnetfoundation.org/campaign-2019/mitchel-sellers.html) with some details regarding my experience on board of directors, but I'll add a bit more detail.

I do believe that having experience working on a Board of Directors, of any type, is incredibly valuable experience to bring to the table.  The interaction between Board Members, the communication skills, and the overall logistics of working with organizations in the capacity of a BOD member can be very challenging.  It is a great way to grow your skillsets, but it can take some time to become effective.

## DMACC Alumni Association
This was my first experience working on a board of directors. The alumni association works to promote the successes of former DMACC students and the professional/personal/civic accomplishments that have been obtained by graduates. During my tenure on the board from 2011-2018 I served as President for 2 years, vice president for 2 years, and a member the rest of the time.

This organization has been focused on outreach to former students to keep them engaged in the college in any capacity; becoming an instructor, donating time/money, mentorship, etc.  As part of this we worked to morph fundraising efforts from more "dated" methods to programs that would have better success with the younger demographic.  We offered [annual awards](https://dmacc.edu/alumni/Pages/past-alumni.aspx) to highlight key successes and ensured that we could offer approximately $20-25k per year in scholarships.

## DNN Association

I joined the board of directors of this organization in early 2018 and assumed the role of Treasurer during the finalization of the organizations application for 501c3 desgination.  This organization is dedicated to providing educational events for users of the DNN Open Source Project (A .NET Foundation Project).  

Assuming the position on this board has been enlightening as many policies and procedures needed to be established and working to coordinate the speaker selection process has educated me on great gaps that exist between speakers that are willing to speak, and groups looking for speakers.  A true parallel to some of the gaps that I believe involvement with the .NET Foundation board can be improved upon.

## Cirrus Owners and Pilots Association (COPA)

I offically joined this board in August of 2018 and serve as its IT Director.  The organization is a 501c7 Social Organization with 5,000+ members and is dedicated to promoting safe piloting/usage of Cirrus Aircraft with a worldwide audience.

This orgnaization has provided me with an incredible growth opportunity as I'm stepping into an organization thas has been around since the early 2000's, however, they have no standards in place.  We are working to modernize a .NET based forum solution that is incredibly active (300k page views/month, > 1.5 million posts). With a truely distributed board (we span 7 time zones) my communication skills and leadership continue to be honed each week.

## In Summary

I feel that every organization that I am invovled with is a way for me to take past experience and apply it to the future.  With each of the above organization I've have a unique experience to work with organzations with different goals and purposes.  Each have affored me the opportunity to grow both as a person and professionally.

I feel greatly that my experience; with boards of directors, community engagement, and business make me a unique candidate and I feel that I could be effective in keeping great communciation chains open as we work to expand the power of the .NET Foundation.

**Comment by SeanKilleen** [comment]
I have no specific Board of Directors experience beyond serving as a member of the Board of Governors for a college association I'm a member of. However, I do work with groups of professionals in many contexts and am continually forming consensus, bringing folks along on a vision (technical or otherwise) for projects both internally and externally, and motivating groups of individuals to achieve a larger impact.

One of my favorite professional moments to date was someone I respect turning to me after a difficult client engagement and saying "....you saw this in your head a year ago. You've been playing chess the whole time!" With large movements that we care about, we often need to take a long view and set ourselves up for success. I have a history of doing that in other contexts and would be delighted to apply that to the .NET Foundation's mission. I pride myself on being a leader beyond the technical sense and in motivating teams to great outcomes.

**Comment by SaraJo** [comment]
Hello! This is a great question. I recruited and managed the non-profit board at **Girl Develop It**. I also managed our advisory board at **Jewelbots**, including specific advisors around open source. 

I've always found it to be a challenging position that keeps me on my toes and doing my best work.

**Comment by stevejgordon** [comment]
Thanks @ahoefling for a very valid question. I have no prior board experience and honestly I'm generally a fairly quiet and socially awkward person (something I'm always working to change). These things were very much in my mind when I was deciding whether I should stand for a seat on the board. After some careful consideration I released that this is exactly why the process is so important. We're seeking to represent and champion diversity and inclusion in our community. In my opinion, we need a board that includes people with diverse interests, skills and social backgrounds if we are to achieve that.

In terms of skills I possess which I believe make me a strong candidate:

- I am passionate about teaching others and sharing knowledge. I believe an important role of the board is helping to raise awareness of the foundation, it's goals and objectives.  
- I am great at listening and considering the views of others. An important aspect of the board is not just acting, but listening. Board members should be a conduit for representing the voice of the wider community in board discussions and decisions. 
- I'm a hard worker and a quick learner. A little over 1.5 years ago I founded a local .NET meetup where I have developed the skills to manage and organise all of our events.
- I have previous experience of managing a team which provided end-user support to over 7,000 users in the EMEA region. In that role I was responsible for customer satisfaction, financial accountability and driving cost reduction and efficiencies in our delivery.

**Comment by rdiazconcha** [comment]
(English)

Hi Andrew!

Like you, I think having previous experience is not required. In fact, I am completely convinced that the most important requirement to be on the Board of Directors of the .NET Foundation is to have **EMPATHY**, in order to truly understand the social, cultural and technical background of each of the members and the entire .NET community in general, to be able to help in the best way so the community offers value to the people.

Having said that, my experience is based on belonging to the Board of Directors of my own company (in the healthcare vertical), in addition to having led in the past several education and online training initiatives. Today, I manage the .NET Latam community with more than 8500 people on Facebook.

(Español)

¡Hola Andrew! 

Igual que tú, creo que tener experiencia previa no es requerido.  De hecho, estoy completamente convencido que el requerimiento más importante para estar en la Junta Directiva de la .NET Foundation es tener **EMPATÍA** para conocer las dinámicas sociales, culturales y técnicas de cada uno de los miembros y de toda la comunidad .NET en general, para poder ayudar de la mejor manera a que la comunidad sea de valor y crezca sanamente.

Dicho esto, mi experiencia se basa en pertenecer a la Junta Directiva de mi propia empresa, además de haber dirigido en el pasado diversas iniciativas de educación y entrenamiento en línea.  Hoy en día, administro la comunidad .NET Latam con más de 8500 personas en Facebook.

**Comment by daveaglick** [comment]
While I don't have any experience serving on a Board of Directors, I'm extremely engaged in somewhat similar leadership roles within my local communities.

* I'm currently the chair of my local scout committee which includes leading a team of 8 - 10 committee members as well as teams of volunteers to support over 60 school-age youth.
* I'm currently the treasurer for my school's Parent-Teacher Association (PTA). This includes handling a yearly budget of approximately $40k and working with volunteer teams for events that have a monetary component.
* I've previously served in various positions on my neighborhood Homeowner's Associations including Treasurer and Vice President.

I’m really looking forward to applying the same energy I give my family and local communities to my global community of open source peers in a more formal capacity.

**Comment by MarcBruins** [comment]
Thanks for the question @ahoefling. 

Short answer: I have no experience as a board member, i also don't think this is a blocker.

Sure experience counts, but a fresh view is also very welcome once in a while. As an open source maintainer, I want to bring this experience to the table. I've been a maintainer on MvvmCross and now maintainer of NuKeeper. This could be seen as a leadership role in the open source space.

**Comment by IEvangelist** [comment]
This is a great question @ahoefling!

### Relevant leadership roles and experience.
| Role  | Activity | Duration |
|--:|---|---|
| Technical Steering / Speaker Selection  |  MKE DOT NET (Developer Conference)  | 3 Years |
| Primary Organizer | Cream City Code (Developer Conference) | 2 Years |
| Board Member / Track Lead |  The Fall Experiment (Conference) | 1 Year |

I'm not sure if this helpful or not, but it is what I have to offer. I think that being involved in these sorts of roles is relevant regardless. While organizationally things will be different, the core factors are seamlessly transferable.

**Comment by sbwalker** [comment]
I served on the Board of Directors for the OuterCurve ( Codeplex ) Foundation which was Microsoft's first attempt to create a non-profit open source foundation. I served on the board with a number of folks from Microsoft as well as from the community. I gained a lot of valuable experience which is directly  applicable to the .NET Foundation.

I served on the Board of Directors for DNN Corp - guiding the company through 3 rounds of venture capital financing. I served on the board alongwith the CEO of DNN Corp, Vivek Mehra from August Capital, Tim Guleri from Sierra Ventures, Chris Cooper from Pelion Venture Partners, and Larry Augustin from SugarCRM. I learned a great deal about commercial boards, fiduciary responsibility, etc...

I served as Chairman of the Advisory Council for the .NET Foundation since 2014. I authored the original Advisory Council charter for the foundation. I have worked closely with the Executive Director of the .NET Foundation for the past 5 years to help him execute the directions from the Board.

I have also served on a number of HOA boards and youth athletic boards.

**Comment by jessicabrentnall** [comment]
Hi @ahoefling. 

Fantastic question. 

I have no experience working on a Board of Directors but I think I can draw on a number of my experiences to adapt and learn to the role. Here are a few examples:

- At the very beginning of my career, I started a diversity meetup. I had been attending meetups to determine if technology was an industry I wanted to join and what aspects I liked. During this short time, I noticed the lack of diversity in meetups and wanted to encourage more people to attend these events which I really enjoyed. This taught me a lot about all aspects of running an event: listening to attendees needs, organising the financial and political aspects of running an inclusive event, making sure the code of conduct was adhered to and more. Over the course of a year, it had a big impact on the attendance of other events and is now itself, one of the most highly attended meetups in the area.

- I've worked closely with local universities from creating student projects to be worked on, managing those projects as both a developmental opportunity for the staff in the companies I've worked for and students and providing mentorship. This involves communicating and organising across all levels.

- I'm co-founding a conference with my partner this year, which includes running it as a not-for-profit company. This requires a lot of transferrable skills from the paperwork, through to communications about the event to different audiences.

These are but a few examples of where I may draw from experience, but like everything I take on, I plan to learn from those around me. I always put in my best efforts when taking on challenges and have demonstrated the organisational, interpersonal and emotional skills to take on a responsibility like this one. 

By no means do I think it will be easy, but the things worth doing are rarely easy.
